simple-calc/modules/Getting Started/README.md

```diff
@@ -82,7 +82,7 @@ To continue, please select one of the following options:
 <div class="row">
 
 <div class="col-md-6 col-xs-12 ">
-<a href="MVUX-XAML\01-Build your first Project\README.md">
+<a href="..\MVUX-XAML\01-Build your first Project\README.html">
```

Comment:
Is it possible to use UIDs here instead?

Reply:
@jeromelaban / @nickrandolph / @kazo0 
I will merge this PR so we can have the fix in the documentation for now and @nickrandolph can answer / test after regarding the UIDs